src/icons/fileIcons.ts

```diff
@@ -1712,5 +1712,6 @@ export const fileIcons: FileIcons = {
     },
     { name: 'rome', fileNames: ['rome.json'] },
     { name: 'cypress', fileNames: ['cypress.json', 'cypress.env.json'] },
+    { name: 'ndst', fileExtensions: ['.ndst.yml', '.ndst.yaml', '.ndst.json'] },
```

Comment:
Hi @imartingon, the dots before the values in the `fileExtensions` list are not needed. Could you please remove them?

It should look like this:

```ts
{ name: 'ndst', fileExtensions: ['ndst.yml', 'ndst.yaml', 'ndst.json'] },
```

Reply:
Cool, thanks you.